src/lib/metaphysics.ts

```diff
@@ -27,6 +27,15 @@ export function metaphysics<T>(
       }
     })
     .then<T>(response => response.json())
+    .then(json => {
+      if (json.errors) {
+        errors.forEach(console.error)
```

Comment:
This is going to go to Sentry, which really indicates server-side errors, but might be good to know if the app suffers from specific endpoints not working well?